convert2rhel/actions/__init__.py

```diff
@@ -719,28 +754,38 @@ def run_actions():
     # Run the Actions in system_checks and all subsequent Stages.
     results = system_checks.run()
 
-    # Format results as a dictionary:
-    # {
-    #     "$Action_id": {
-    #         "messages": [
-    #             {
-    #                 "level": int,
-    #                 "id": "$id",
-    #                 "message": "" or "$message"
-    #             },
-    #         ]
-    #         "result": {
-    #             "level": int,
-    #             "id": "$id",
-    #             "message": "" or "$message"
-    #         },
-    #     },
-    # }
-    formatted_results = {}
-    for action in itertools.chain(*results):
-        msgs = [msg.to_dict() for msg in action.messages]
-        formatted_results[action.id] = {"messages": msgs, "result": action.result.to_dict()}
-    return formatted_results
+    return parse_action_results(results)
+
+
+def run_post_actions():
+    """
+    Run all of the post-ponr Actions.
+
+    This function runs the Actions that occur after the Point of no Return.
```

Comment:
```suggestion
    This function runs the Actions that occur after the Point of no Return.
```
Maybe I would keep just one of those sentences. They say the same from my point of view, but the second is better. And if removing one, then I would remove the same under `run_pre_actions`